serverless.yml

```diff
@@ -1,6 +1,6 @@
 service: biztechApp
-tenant: ianmh
-app: biztechapp
+org: biztechdev
```

Comment:
Can you revert this and see if it works again?

Reply:
works after reverting!